src/control/cmd/dmg/pretty/storage_query.go

```diff
@@ -264,6 +266,9 @@ func printTierUsageTable(hsm control.HostStorageMap, tierRoles []storage.BdevRol
 	sort.Ints(ranks)
 
 	for _, rank := range ranks {
+		if rank == int(ranklist.NilRank) {
+			continue
+		}
```

Comment:
Huh. Under what circumstances can this happen? Is it only if an engine is both down and has never started before?

Reply:
During scan for off-line devices (when engine is stopped) usage stats cannot be gathered correctly as dRPC is unavailable. The scan for the off-line devices is performed over the SPDK bindings and the results returned contain NilRank as the engine is stopped and therefore it doesn't have an active rank. As the usage statistics for a stopped rank cannot be calculated without dRPC the NilRank entries  results are not displayed.